dali/base/dasess.cpp

```diff
@@ -1954,11 +1954,17 @@ bool registerClientProcess(ICommunicator *comm, IGroup *& retcoven,unsigned time
                 retcoven = deserializeIGroup(mb);
                 if (!mySessionId)
                     throw deserializeException(mb);
+                if (lastNextLog) // see below, true if has been round timeout loop. Only output confirmation, if have issued 'Failed to connect ..' messages
```

Comment:
Minor - I might argue whats the harm in always logging this ?  Too much logging ?